frontend/src/features/map/layers/styles/isForbiddenPeriod.ts

```diff
@@ -8,22 +10,33 @@ import type Feature from 'ol/Feature'
 export function isForbiddenPeriod(feature: Feature | undefined, currentDate: Dayjs) {
   const currentWeekDayDigit = currentDate.day()
 
-  const fishingPeriodString = feature?.get('fishing_period')
-  if (!fishingPeriodString) {
+  const fishingPeriodValue = feature?.get('fishing_period') || feature?.get('fishingPeriod')
+  if (!fishingPeriodValue) {
     return false
   }
 
-  const fishingPeriod = JSON.parse(fishingPeriodString) as FishingPeriod
+  const fishingPeriod = isObject(fishingPeriodValue)
+    ? (fishingPeriodValue as FishingPeriod)
+    : (JSON.parse(fishingPeriodValue) as FishingPeriod)
+
   switch (fishingPeriod.authorized) {
     /**
-     * It is forbidden when
-     *  - isAlwaysAuthorized is false AND
+     * If the dates, weekdays and dateRanges are not set, the period won't be forbidden by default.
+     *
+     * If they are set, the period will be forbidden when
      *  - hasAllowedRange is false AND
      *  - hasAllowedDate is false AND
      *  - hasAllowedWeekDay is false
      */
     case true: {
-      const isAlwaysAuthorized = fishingPeriod.always
+      const isAlwaysAuthorized =
+        removeNullValues(fishingPeriod.dates).length === 0 &&
```

Comment:
I would have made a small global util (i.e.: `isNullishEmpty()`) for that and write it like that:

```tx
const isAlwaysAuthorized = isNullishEmpty([...fishingPeriod.dates, ...fishingPeriod.weekdays, ...fishingPeriod.dateRanges])
```

Reply:
If it's used in another part of the app, I'll do it, for the time being I don't think we need an util.